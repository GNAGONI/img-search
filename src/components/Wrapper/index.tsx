import { styled } from '@mui/material/styles';

type Props = {
  orientation: 'row' | 'column';
  children: React.ReactNode;
};

const StyledWrapper = styled('div')<Props>(({ orientation }) => ({
  display: 'flex',
  flexDirection: orientation,
  justifyContent: 'center',
  alignItems: 'center',
  height: '100%',
}));

const Wrapper = (props: Props) => {
  return (
    <StyledWrapper orientation={props.orientation} >
      {props.children}
    </StyledWrapper>
  );
};

export default Wrapper;