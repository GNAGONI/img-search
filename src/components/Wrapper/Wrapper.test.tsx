import renderer from 'react-test-renderer';
import Wrapper from './index';

it('renders correctly Wrapper', () => {
  const tree = renderer
    .create(
      <Wrapper orientation='row'>
        <></>
      </Wrapper>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});