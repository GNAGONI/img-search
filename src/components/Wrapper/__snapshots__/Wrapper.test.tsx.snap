// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Wrapper 1`] = `
<div
  className="css-146mucv"
  orientation="row"
/>
`;
